src/core/PrivacyFilter/PrivacyFilter.tsx

```diff
@@ -5,30 +5,38 @@ import patternWaves from "/static/images/pattern-waves.png";
 interface PrivacyFilterProps {
   editable: boolean;
   onToggleEditable: () => void;
-  className?: string;
+  options?: {
+    className?: string;
+    description?: string;
+    buttonText?: string;
+  };
 }
 
-export const PrivacyFilter: FunctionComponent<PrivacyFilterProps> = ({ editable, onToggleEditable, className }) => (
-  <div
-    className={className ? className : "print:hidden bg-cover bg-cerulean text-white rounded-lg p-4 mb-8"}
-    style={{ backgroundImage: `url(${patternWaves})` }}
-  >
-    <div className="container">
-      <div className="md:flex items-center">
-        <div className="grow mb-4 md:mb-0 mr-0 md:mr-4">
-          <h3 className="font-normal">The document allows fields to be selectively disclosed.</h3>
-          <p>
-            Remove sensitive information on this document by clicking on the edit button. Downloaded document remains
-            valid.
-          </p>
+export const PrivacyFilter: FunctionComponent<PrivacyFilterProps> = ({ editable, onToggleEditable, options }) => {
+  const className = options?.className
+    ? options.className
+    : "print:hidden bg-cover bg-cerulean text-white rounded-lg p-4 mb-8";
+  const description = options?.description
+    ? options.description
+    : `Remove sensitive information on this document by clicking on the edit button. Downloaded document remains
+  valid.`;
+  const buttonText = options?.buttonText ? options.buttonText : "Edit Document";
```

Comment:
how about accessing `options` via this:

```
const { className = "your default values", buttonText = "Edit Document" } = options;
```

Reply:
when I do the above it errors out because options is not mandatory.
how is smth like this?

``` 
const defaultOptions = { 
    className: "print:hidden bg-cover bg-cerulean text-white rounded-lg p-4 mb-8",
    description: `Remove sensitive information on this document by clicking on the edit button. Downloaded document remains valid.`,
    buttonText: "Edit Document" }
const { className, description, buttonText } = options ?? defaultOptions
  ```